gadgets/trace_malloc/program.bpf.c

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: GPL-2.0
+/* Copyright (c) 2024 The Inspektor Gadget authors */
+
+#include <vmlinux.h>
+#include <bpf/bpf_helpers.h>
+#include <bpf/bpf_tracing.h>
+#include <gadget/buffer.h>
+#include <gadget/macros.h>
+
+struct event {
+	__u32 pid;
+	__u64 addr;
+	bool is_malloc;
```

Comment:
In the next PR, could you add the `comm` field?

Reply:
btw, I'm looking for how to print the `addr` in hex format, but I didn't find any examples. This is a note that I should look into it later.

Reply:
cc @flyth

Reply:
> I'm looking for how to print the addr in hex format, but I didn't find any examples. This is a note that I should look into it later.

We don't support it yet. Would you mind opening an issue? I suppose we can implement it by having a new attribute that indicates it must be printed using hex format.

Reply:
Opened issue #2589